eggplant_project/settings/base.py

```diff
@@ -15,6 +15,10 @@
 from django.utils.translation import ugettext_lazy as _
 from django.contrib.messages import constants as messages_constants
 
+COOP_NAME = 'KBHFF'
```

Comment:
I would much prefer a pattern that defines settings and their defaults in the app. Like so:

`eggplant.settings`:

    from django.conf import settings
    COOP_NAME = getattr(settings, 'EGGPLANT_COOP_NAME', 'default_value')

Then in the context preprocessor, you use eggplant.settings.COOP_NAME.